dkist/dataset/tiled_dataset.py

```diff
@@ -333,10 +342,16 @@ def plot(
                 if swap_tile_limits in ["y", "xy"]:
                     ax.invert_yaxis()
 
+                if col == row == 0:
+                    xlabel, ylabel = self._get_axislabels(ax)
                 ax.set_ylabel(" ")
                 ax.set_xlabel(" ")
```

Comment:
These lines would be better off using the wcsaxes API for consistency. https://docs.astropy.org/en/stable/visualization/wcsaxes/ticks_labels_grid.html#axis-labels